examples/seismic/abc_methods/02_damping.ipynb

```diff
@@ -0,0 +1,1257 @@
+      "text/plain": [
+       "<Figure size 1152x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "graph2damp(D0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like the velocity function $c(x,z)$, the damping function $\\zeta(x,z)$ is constant in time. Therefore, the damping function will be a second-order *Function* in space, which uses points of the non-staggered type and which we will evaluate with the D0 array. The symbolic name *damp* will be assigned to this field."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "damp = Function(name=\"damp\",grid=grid,space_order=2,staggered=NODE,dtype=np.float64)\n",
+    "damp.data[:,:] = D0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The expressions for the acoustic equation with damping can be separeted between the white and blue regions.\n",
+    "\n",
+    "Translating these expressions in terms of an *eq* that can be inserted in a Devito code, we have that in the white region the equation takes the form:\n",
+    "\n",
+    "- eq1 = u.dt2 - vel0 * vel0 * u.laplace,\n",
+    "\n",
+    "and in the blue region we have the following equation:\n",
+    "\n",
+    "- eq2 = u.dt2 + vel0 * vel0 * damp * u.dtc - vel0 * vel0 * u.laplace.\n",
+    "\n",
+    "Here *u.dtc* represents the centered derivative with respect to the variable $t$ for the field *u*. Then, we set the two pdes for the two regions\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pde0 = Eq(u.dt2 - u.laplace*vel0*vel0)\n",
```

Comment:
Why write as `vel0*vel0`?

Reply:
It was just a way of writing the expression. I tried to replace in all the notebooks this expression for vel**2. If this type of writing occurs again, I can correct it quickly with the instructions.